.changes/unreleased/Fixes-20241008-122635.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Dynamic Iceberg table DDL catalog parameter fixes.
```

Comment:
I would consider making this changelog entry specific to `base_location_subpath`.